src/Command/BindDeputiesToDistrictsCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\District;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Serializer\Encoder\CsvEncoder;
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Serializer;
+
+class BindDeputiesToDistrictsCommand extends Command
+{
+    protected static $defaultName = 'app:deputy-districts:bind';
+
+    private $em;
+    private $decoder;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    public function __construct(EntityManagerInterface $em)
+    {
+        $this->em = $em;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->addArgument('file', InputArgument::REQUIRED, 'CSV file of deputies with their managed districts to load');
+        $this->setDescription('Bind deputies to districts. Attention! If district has a deputy it will be changed without warning.');
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+        $this->decoder = new Serializer([new ObjectNormalizer()], [new JsonEncoder(), new CsvEncoder()]);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if (!is_readable($file = $input->getArgument('file'))) {
+            $this->io->error("$file is not a file");
+
+            return 1;
+        }
+
+        $this->io->text('Start binding deputies to districts');
+
+        $deputies = $this->decoder->decode(file_get_contents($file), 'csv', [CsvEncoder::DELIMITER_KEY => ';']);
+        $this->io->title(sprintf('Starting bind deputies to districts: %s deputies are about to be binded.', count($deputies)));
+
+        $i = 0;
```

Comment:
You don't really need this variable.

Reply:
I use it here https://github.com/EnMarche/en-marche.fr/pull/2512/files#diff-ae51f37302ff7c267728b0258b517d83R93